Jalasoft.TeamUp.Projects.ProjectsException/ProjectsException.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
```

Comment:
using should be inside of namespace
I thing you need to add stylecop to this new project